packages/react/src/components/Tile/Tile.js

```diff
@@ -15,7 +15,7 @@ import {
   ChevronDown16,
 } from '@carbon/icons-react';
 import { keys, matches } from '../../internal/keyboard';
-import uid from '../../tools/uniqueId';
+import setupGetInstanceId from '../../tools/setupGetInstanceId';
```

Comment:
is this one of our helpers? Idk anything about it, so just want to make sure

Reply:
Yeah! I think it replaced uid as the new way to quickly make something unique, according to @joshblack